packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -159,9 +159,11 @@ contract OptimismPortal is Initializable, ResourceMetering {
             from = AddressAliasHelper.applyL1ToL2Alias(msg.sender);
         }
 
+        bytes memory opaqueData = abi.encode(msg.value, _value, _gasLimit, _isCreation, _data);
```

Comment:
`opaqueData` was the term used in the spec, but IMO `depositData` would be more reader friendly.